ui/prebuild.sh

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env bash
+
+configFile='./public/config.json'
+tmpFile='./public/config.json.tmp'
```

Comment:
@shwstppr could relative paths cause any issue? For example script being call outside of the folder?

Reply:
@rhtyd made the change